src/content/en/updates/2020/07/referrer-policy-new-chrome-default.md

```diff
@@ -0,0 +1,237 @@
+requests from **another origin** to your site (more specifically the path and/or query string) AND
+this origin uses the browser's default referrer policy (i.e. it has no policy set).
+
+### If this impacts your site, consider alternatives
+
+If you're using the referrer to access the full path or query string for requests to your site, you
+have a few options:
+
+- Use alternative techniques and headers such as `Origin` and `Sec-fetch-Site` for your CSRF
+  protection, logging, and other use cases. Check out [Referer and Referrer-Policy: best
+  practices](https://web.dev/referrer-best-practices).
+- You can align with partners on a specific policy if this is needed and transparent to your users.
+  Access control—when the referrer is used by websites to grant specific access to their resources
+  to other origins—might be such a case although with Chrome's change, the origin will still be
+  shared in the `Referer` Header (and in `document.referrer`).
+
+Note that most browsers are moving in a similar direction when it comes to the referrer (see browser
+defaults and their evolutions in [Referer and Referrer-Policy: best
+practices](https://web.dev/referrer-best-practices/#default-referrer-policies-in-browsers).
+
+### Implement an explicit, privacy-enhancing policy across your site
+
+What `Referer` should be sent in requests **originated** by your website, i.e. what policy should
+you set for your site?
+
+Even with Chrome's change in mind, it's a good idea to set an **explicit, privacy-enhancing policy**
+like `strict-origin-when-cross-origin` or stricter right now.
+
+This protects your users and makes your website behave more predictably across browsers. Mostly, it
+gives you control —rather than having your site depend on browser defaults.
+
+Check [Referer and Referrer-Policy: best practices](https://web.dev/referrer-best-practices) for
+details on setting a policy.
+
+### About Chrome enterprise
+
+The Chrome enterprise policy
+[`ForceLegacyDefaultReferrerPolicy`](https://cloud.google.com/docs/chrome-enterprise/policies/?policy=ForceLegacyDefaultReferrerPolicy)
+is available to IT administrators who would like to force the previous default referrer policy of
+`no-referrer-when-downgrade` in enterprise environments. This allows enterprises additional time to
+test and update their applications. 
+
+This policy will be removed in Chrome 88.
+
+
+## Send feedback
+
+Do you have feedback to share or something to report? Share feedback on [Chrome's intent to
+ship](https://groups.google.com/a/chromium.org/forum/#!topic/blink-dev/lqFuqwZDDR8), or tweet your
+questions at [@maudnals](https://twitter.com/maudnals).
+
+_With many thanks for contributions and feedback to all reviewers - especially Kaustubha Govind,
+David Van Cleve, Mike West, Sam Dutton, Rowan Merewood, Jxck and Kayce Basque._
+
+{% include "web/_shared/helpful.html" %}
+
+{% include "web/_shared/rss-widget-updates.html" %}
```

Comment:
I would move these two down below the resources section.

Reply:
Sounds good, done!